esx_service/vmdk_ops_test.py

```diff
@@ -204,33 +204,6 @@ def testAttachAsOpts(self):
 
     @unittest.skipIf(not vsan_info.get_vsan_datastore(),
                     "VSAN is not found - skipping vsan_info tests")
-    def testPolicyUpdate(self):
```

Comment:
This test creates a vmdk inside dockvols folder, assign a good policy to it and then update the policy with bad content. In this case, it tries to list the vmdks associated with the good policy and set the updated policy to it. The test expects the update to fail because of the bad content. In this process, when the vsan_policy update api is invoked, no vmdks are listed because the vmdks are created inside tenant uuid named sub directories as per multi tenancy design. As no vmdk that would be affected are returned, this test case wasn't throwing an error.
I haven't removed it, but moved it to the test suite where tenant based vmdks are created.
Hence it properly lists the vmdk files, tries to update it and fails appropriately because of the bad content of policy